### DISCUSSION TITLE
Documentation to interpret the Summary report

### DISCUSSION THREAD

**Discussion by Cryptinterested** [artilleryio/artillery#1557] [Artillery]
When performing my performance tests, I get a Summary report but I struggle to find an exhaustive documentation to explain each section of the report.

This is my final report:

`All VUs finished. Total time: 4 minutes, 30 seconds

--------------------------------
Summary report @ 21:20:11(+0200)
--------------------------------

errors.ETIMEDOUT: .............................................................. 184279
http.codes.404: ................................................................ 444
http.request_rate: ............................................................. 746/sec
http.requests: ................................................................. 184723
http.response_time:
  min: ......................................................................... 58
  max: ......................................................................... 6688
  median: ...................................................................... 127.8
  p95: ......................................................................... 1002.4
  p99: ......................................................................... 2671
http.responses: ................................................................ 444
vusers.completed: .............................................................. 444
vusers.created: ................................................................ 184723
vusers.created_by_name.Get header: ............................................. 184723
vusers.failed: ................................................................. 184279
vusers.session_length:
  min: ......................................................................... 192.5
  max: ......................................................................... 9987.4
  median: ...................................................................... 497.8
  p95: ......................................................................... 8024.5
  p99: ......................................................................... 9416.8
`

Is `http.request_rate` a mean or a max? 
Can `http.responses:` be different from `vusers.completed: `?

**Comment by hassy** [comment]
@Cryptinterested `http.request_rate` is average over the reporting period. Yes `http.responses` is usually different, because a scenario typically makes more than one HTTP request. They would be the same if every VU makes only one request.

**Reply by Cryptinterested** [reply]
Thanks for your answer. Is it possible to get the max request_rate?

**Reply by Cryptinterested** [reply]
Any news on that?

**Reply by hassy** [reply]
There's no way to track max request rate right now. What way to measure it would be most useful for your use case?

**Reply by Cryptinterested** [reply]
I would say the last moment when the remote server answers to all my requests. I would define that as a maximum.
